<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1433</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>60</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(177, 146, 255);</string>
    </property>
    <property name="text">
     <string>Consultation</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>100</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(61, 210, 255);</string>
    </property>
    <property name="text">
     <string>Contr√¥le</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>140</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(42, 42, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Lecture des Analyses</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>180</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 199, 68);</string>
    </property>
    <property name="text">
     <string>Lecture de Scanner</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>220</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(191, 151, 117);</string>
    </property>
    <property name="text">
     <string>Polygraphie</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>260</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 226, 196);</string>
    </property>
    <property name="text">
     <string>Lecture de Polygraphie</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>300</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(191, 180, 176);</string>
    </property>
    <property name="text">
     <string>Prick Test</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>340</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 185, 172);</string>
    </property>
    <property name="text">
     <string>Spirometrie</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widgetCentral" native="true">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>9</y>
      <width>1201</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1051</width>
       <height>771</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>50</y>
        <width>751</width>
        <height>691</height>
       </rect>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>19</number>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>08h00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>08h30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>09h00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>09h30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10h00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10h30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11h00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11h30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12h00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12h30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13h00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13h30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14h00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14h30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15h00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15h30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>16h00</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>16h30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>17h00</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>01</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>02</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>03</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>04</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>05</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>06</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>07</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButtonPrendreRDV">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>80</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Prendre un RDV</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDateToday">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 24pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Janvier 2020</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1433</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
